render/src/utils.rs

```diff
@@ -78,7 +78,7 @@ pub fn glue_tables_to_jpeg<'a>(
 
 /// Removes potential invalid JPEG data from SWF DefineBitsJPEG tags.
 /// These bytes need to be removed for the JPEG to decode properly.
-pub fn remove_invalid_jpeg_data(data: &[u8]) -> Cow<[u8]> {
+pub fn remove_invalid_jpeg_data(data: &[u8]) -> Cow<'_, [u8]> {
```

Comment:
That's an unrelated change, you don't have to include it. I know that beta lints complain about it, but we can ignore them for now.